pkg/api/dtos/plugins.go

```diff
@@ -51,6 +51,7 @@ type PluginListItem struct {
 	AngularDetected bool                    `json:"angularDetected"`
 	IAM             *pfs.IAM                `json:"iam,omitempty"`
 	APIVersion      string                  `json:"apiVersion"`
+	Managed         bool                    `json:"isManaged,omitempty"`
```

Comment:
I wonder if the wording "managed" is the best choice here, the term is already in used to identify plugins that use automatic instance management. Can we say `remote`  or has that term also a different meaning?

Reply:
The term is tricky alright - happy to iterate on that. ideally we want the same across backend and frontend. `Managed` right now is backend plugins that are auto started and kept alive by Grafana, which is compeltely different 😞 The fact that it's remote here is more a implementation detail, it's about having a plugin that Grafana will manage/operate for you, similar to a Core plugin.

Reply:
Since this is internal (not user-facing) I think it's more readable (maybe it's just for me) to use `remote` here and in the backend. Another option could be `ManangedRemotely` if it's not too long?

Reply:
I think managed make sense. We already have old config for user management referring to external and managed, https://github.com/grafana/grafana/blob/0072e4a92d896df343d9586522d6f7533773da78/conf/defaults.ini#L505-L508

Reply:
Also this one which I would like to remove eventually https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#plugin_admin_external_manage_enabled. Not sure how much it's relied on now that we have in-catalog installs.

Reply:
+1 for managed as it reads the most clear to me.